.werft/build.ts

```diff
@@ -128,7 +147,12 @@ export async function build(context, version) {
         cleanSlateDeployment,
         installEELicense,
         withObservability,
-    }));
+    }
+    werft.log("job config", JSON.stringify(jobConfig));
+    werft.rootSpan.setAttributes(Object.fromEntries(Object.entries(jobConfig).map((kv) => {
+        const [key, value] = kv
+        return [`werft.job.config.${key}`, value]
+    })))
```

Comment:
This is super nice!!!

That means that we can aggregate and measure latency/success rate by configuration, right?

e.g. we could notice that success rate is X% when `withObservability == true`  and X-10% when `withObservability == false`

Reply:
Could we also add the current branch as an attribute? I think it would be useful since one single branch is often an outlier for failures, e.g. when the developer actually broken the pipeline, his/her branch will be the outlier

Reply:
Yeah, that's right 🎉   right now it is only added to the root span, so we can't get the aggregate by phase / slice, but I think that's okay for now.

E.g. here's the success rate grouped by "clean slate deployment".

![Screenshot 2021-11-03 at 13 46 35](https://user-images.githubusercontent.com/83561/140062510-06b87023-80c8-4330-80f5-7e53605dd9cf.png)

I suspect this will be super helpful for bubble-up debugging where look through all the config options easily and find if any specific settings cause errors ☺️

I added the branch under `werft.job.config.branch`  - thanks for the suggestion!